lib/_stream_readable.js

```diff
@@ -714,7 +714,7 @@ function nReadingNextTick(self) {
 
 // pause() and resume() are remnants of the legacy readable stream API
 // If the user uses them, then switch into old mode.
-Readable.prototype.resume = function() {
+Readable.prototype.resume = function _resume() {
```

Comment:
Maybe call this one `resume` and rename the local function to `resumeReadable` ?